lib/helpers.js

```diff
@@ -111,19 +111,20 @@ export const createProfile = async ({
   const {id: profileId} = await profileManager.createProfile(profileOptions);
 
   // update profile and profile agent user with content
-  const {accessManager, profile, profileAgent} = await profileManager
-    .getAccessManager({profileId});
+  const {
+    accessManager, profile: profileToUpdate, profileAgent
+  } = await profileManager.getAccessManager({profileId});
   const [user] = await Promise.all([
     profileAgentContent && _updateProfileAgentUser(
       {accessManager, profileAgent, profileAgentContent}),
     profileContent && _updateProfileUser(
       {accessManager, profileId, profileContent})
   ]);
-
+  let profile = profileToUpdate;
   // if profile content was provided, kick off request to update cache
   if(profileContent) {
     // set `useCache: false` to fetch a fresh copy
-    profileManager.getProfile({id: profileId, useCache: false})
+    profile = await profileManager.getProfile({id: profileId, useCache: false})
```

Comment:
So this isn't the right fix for two reasons:

1. It introduces an `await` which is intentionally avoided here because it increases down profile creation latency to an unacceptable level.
2. Even if the `await` were to be accepted, it still has a `.catch()` here that would cause the profile to become an erroneous value if an error occurred.

So, we don't want to do the `await` here. If `profileContent` was given, we could instead assume that it will, in fact, be integrated into the profile and we could manually apply it to the local `profile` copy (however that's supposed to happen). That would be ok, I think.

Reply:
Thank you. I've added those fields to the profile returned here [9e558fa](https://github.com/digitalbazaar/bedrock-web-wallet/pull/28/commits/9e558fa99621e9ad56ffc659b8ec7fca722b6740)